api_layer/source/tests/matches/model.test.js

```diff
@@ -32,6 +32,33 @@ const testGame2 = {
     valid: true,
 };
 
+const testReport1 = {
+    matchID: '09mwuxcenwqiucnapsdfuhc',
+    gameToken: 'Game 1',
```

Comment:
MatchReports should probably be tested in their own files, seeing as they are a new model.